src/shared/domain/case-event-data.model.ts

```diff
@@ -9,4 +9,5 @@ export class CaseEventData {
   event_token: string;
   ignore_warning: boolean;
   draft_id?: string;
+  case_reference?: string;
```

Comment:
Shouldn't case reference be mandatory? i.e. `case_reference` rather than `case_reference?`